Document renaming the journald log folder when changing machine id (#3903)
uyuni-project/uyuni-docs: modules/administration/pages/troubleshooting/tshoot-registerclones.adoc

systemd-machine-id-setup
----

> If the cloned machine also has a folder in [path]``/var/log/journal/`` it also needs to be renamed with the new machine ID.
> Without this, [command]``journalctl`` would not persist any log and [command]``podman logs`` would not show anything.
> 

Reviewer: ```suggestion
. If the cloned machine also has a folder in [path]``/var/log/journal/`` it needs to be renamed accordingly to the new machine ID.
If names do not match, [command]``journalctl`` would not persist any log and [command]``podman logs`` would not show anything.

+

```
I do not understand what "would not persist any log" means in this context. (`enter` or `find` a log entry?)

Author: it means that `journalctl -u` for any systemd service would not given anything: logs would be lost. Same for `podman logs <yourcontainer>` or `mgrpxy logs`: they would return nothing and the containers logs would be lost.
